demo/Cargo.toml

```diff
@@ -14,10 +14,9 @@ shaders = { path = "shaders" }
 distill = { version = "=0.0.3", features = ["serde_importers"] }
 uuid = "0.8"
 type-uuid = "0.1"
-sdl2 = { version = "0.34", features = ["raw-window-handle"] }
 imgui = { version = "0.7", optional = true }
-imgui-sdl2 = { version = "0.14.0", optional = true }
 egui = { version = "0.12", features = ["default_fonts"], optional = true }
+copypasta = { version = "0.7", optional = true }
```

Comment:
It looks like this transitively included `lazy-bytes-cast` which was BPL. 
![image](https://user-images.githubusercontent.com/4888530/120936512-02060680-c6bd-11eb-8abe-78fcc9577827.png)

Reply:
This is also raising another error in `cargo-deny`
![image](https://user-images.githubusercontent.com/4888530/120936563-514c3700-c6bd-11eb-87b7-3c7edcf5239e.png)

Reply:
`xcb` is included by `copypasta`